src/cclib/parser/orcaparser.py

```diff
@@ -583,7 +583,11 @@ def extract(self, inputfile, line):
                     start = (int(start[:-1]), lookup[start[-1]])
                     end = line[10:17].strip()
                     end = (int(end[:-1]), lookup[end[-1]])
-                    contrib = float(line[35:47].strip())
+                    # Coeffients are not printed for RPA, only
+                    # TDA/CIS.
+                    contrib = line[35:47].strip()
+                    if contrib != '':
+                        contrib = float(contrib)
```

Comment:
I am not a fan of this, it's just a temporary placeholder. In the case of no value for `contrib` (which should be a float), should we use `None` or `np.nan`? Then we can check for it in the regression test.